ui-tests/src/test/java/org/openmrs/reference/page/ManageLocationAttributeTypesPage.java

```diff
@@ -0,0 +1,116 @@
+    private static final By DELETE_LOCATION_ICON = By.cssSelector("i.icon-trash.delete-action.right");
+    private static final By CONFIRM_DELETE_LOCATION_BUTTON = By.cssSelector("#purgeLocationAttributeTypeForm button.confirm.right");
+    private static final By CONFIRM_RETIRE_LOCATION_BUTTON = By.cssSelector("#retireLocationAttributeTypeForm button.confirm.right");
+    private static final By CANCEL_RETIRE_LOCATION_BUTTON = By.cssSelector("#retireLocationAttributeTypeForm button.cancel");
+    private static final By CANCEL_DELETE_LOACTION_BUTTON = By.cssSelector("#purgeLocationAttributeTypeForm button.cancel");
+
+    public ManageLocationAttributeTypesPage(Page parent) {
+        super(parent);
+    }
+
+    @Override
+    public String getPageUrl() {
+        return PAGE_URL;
+    }
+    
+    public void goToAddNewLocationAttributeTypeForm() {
+        clickOn(ADD_LOCATION_ATTRIBUTE_TYPE_BUTTON);
+    }
+   
+    public void goToEditLocationAttributeTypeForm() {
+        clickOn(EDIT_LOCATION_ATTRIBUTE_TYPE_ICON);
+    }
+    
+    public void enterAttributeTypeName(String locationName) {
+        findElement(ATTRIBUTE_TYPE_NAME_FIELD).clear();
+        findElement(ATTRIBUTE_TYPE_NAME_FIELD).sendKeys(locationName);
+    }
+
+    public void enterAttributeTypeDescription(String locationDescription) {
+        findElement(ATTRIBUTE_TYPE_DESCRIPTION_FIELD).clear();
+        findElement(ATTRIBUTE_TYPE_DESCRIPTION_FIELD).sendKeys(locationDescription);
+    }
+    
+    public void enterMinimumOccurs(String value) {
+        setText(MIN_OCCURS_FIELD, value);
+    }
+   
+    public void enterMaximumOccurs(String value) {
+        setText(MAX_OCCURS_FIELD, value);
+    }
+  
+    public void selectDatatype(String datatype) {
+        selectFrom(DATA_TYPE_FIELD, datatype);
+    }
+  
+    public void enterDatatypeConfiguration(String datatypeConfiguration) {
+        findElement(DATA_TYPE_CONFIGURATION_FIELD).clear();
+        findElement(DATA_TYPE_CONFIGURATION_FIELD).sendKeys(datatypeConfiguration);
+    }
+   
+    public void selectPreferredHandler(String preferredHandler) {
+        selectFrom(PREFERRED_HANDLER_FIELD, preferredHandler);
+    }
+   
+    public void enterHandlerConfiguration(String handlerConfiguarion) {
+        selectFrom(HANDLER_CONFIGURATION_FILED, handlerConfiguarion);
+    }
+  
+    public void clickOnSaveButton() {
+        clickOn(SAVE_BUTTON);
+    }
+ 
+    public void clickOnCancelButton() {
+        clickOn(CANCEL_BUTTON);
+    }
+    
+    public void purgeLocation() {
+        clickOn(DELETE_LOCATION_ICON);
+        waitForElement(CONFIRM_DELETE_LOCATION_BUTTON);
+        clickOn(CONFIRM_DELETE_LOCATION_BUTTON);
+    }
+
+    public void cancelPurgeLocation() {
+        clickOn(CANCEL_DELETE_LOACTION_BUTTON);
+    }
+    
+    public void retireLocation() {
+        clickOn(By.cssSelector("i.icon-remove.delete-action"));
```

Comment:
Don't you already have a constant for the above?

Reply:
Have done the needful in the commit with id `4da39cc`